### ISSUE TITLE
finetune on small data

### ISSUE THREAD

**Issue by MrYANG23** [keonlee9420/Emotional-FastSpeech2#9]
HI ,i have a issue about finetune ,when FastSpeech2 model trained on big data(ASHELL3), so,when i finetune the pretrained model on samll data(maybe  3 speakers), if I Just change the number speaker in nn.embedding ,and load the pretrained ckpt except the nn.embedding part,like this 
![image](https://user-images.githubusercontent.com/38352877/140884673-7f97fe71-e2d6-49f7-bbd1-b1c6d2221e6a.png)

**Comment by keonlee9420** [comment]
Hi @MrYANG23 , thanks for your attention. That's because the speaker number should be matched with the pre-trained model. You may change the speaker number for fine-tuning on it following [here](https://github.com/keonlee9420/Daft-Exprt/issues/7).

**Comment by MrYANG23** [comment]
thank you ,you have done so many  nice works ,how much number sentences in per-speaker may be get good result  when finetune on small data(3 speakers),I test about per-speaker 50 sentences ,in training the result is ok ,but in val the result is not well.
trian log like this:
![image](https://user-images.githubusercontent.com/38352877/140887885-594e9abb-b167-4f54-bb33-756923f0cceb.png)
val log like this:
![image](https://user-images.githubusercontent.com/38352877/140888112-332751f2-37b7-4ff5-94e6-8d787e4818bf.png)

**Comment by keonlee9420** [comment]
@MrYANG23 sorry for TOO late response.. I think 150 sentences in total is not enough with the base configuration. You can reduce the model size and add some regularization term in loss functions or apply weight decay or dropout.

**Comment by keonlee9420** [comment]
Close due to inactivity.
